nodeutils/save.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Nokia
+// Licensed under the BSD 3-Clause License.
+// SPDX-License-Identifier: BSD-3-Clause
+
+// Package netconf contains netconf-based and functions used in containerlab.
+package nodeutils
+
+import (
```

Comment:
lets bring it back to the netconf package, iso node utils

Reply:
Renamed it back